plugins/dice.js

```diff
@@ -3,9 +3,29 @@ const _ = require('lodash');
 module.exports = (function(){
 
 	let bot,
-		reDice = /([1-9][0-9]{0,2})(d|D)([1-9][0-9]{0,3})((\+|-|\*|\/)([0-9]{1,3}))?/g,
+		reDice = /([1-9][0-9]{0,2})(d|D)([1-9][0-9]{0,3})((\+|-|\*|\/)([0-9]{1,3}))?( *\&(g|G|l|L)t\; *[0-9]*)?/g,
 		reMessage = /(^(\!roll )[1-9][0-9]{0,2})(d|D)([1-9][0-9]{0,3})((\+|-|\*|\/)([0-9]{1,3}))?/g;
 
+	function checkTarget(result, quantifier, target) {
+		let text = '';
+		if (quantifier === '&gt;') {
```

Comment:
would a switch() work better than the ifs? youre only ever checking quantifier right?